src/main/java/org/folio/circulation/domain/MoveRequestService.java

```diff
@@ -78,6 +80,12 @@ private RequestAndRelatedRecords pagedRequestIfDestinationItemAvailable(
     return requestAndRelatedRecords;
   }
 
+  private CompletableFuture<Result<RequestAndRelatedRecords>> validateItemInstanceId(
```

Comment:
Why do we compose it and wrap the result in completedFuture? Doesn't it make more sense to use thenApply and map 'records' with refuseWhenItemToBeMovedIsFromDifferentInstance directly?